pkg/jobrunaggregator/jobtableprimer/generate_jobnames.go

```diff
@@ -223,6 +223,11 @@ func (o *GenerateJobNamesOptions) Run(ctx context.Context) error {
 				continue
 			}
 
+			// OKD jobs are not something we monitor and keep slipping into our disruption data skewing results quite badly.
+			if strings.Contains(curr.Name, "okd") {
```

Comment:
Super nitpick: I kind of like a dash. Is there a chance okd could appear in the middle of a word? Probably not but maybe better to be safe?

Reply:
I couldn't think of a word but you did on slack, I'll add a leading dash just incase it could appear at end, and send in a followup.

Reply:
https://github.com/openshift/ci-tools/pull/3126/files